src/schema/nodes/title.ts

```diff
@@ -30,7 +30,7 @@ export interface TitleNode extends ManuscriptNode {
 }
 
 export const title: NodeSpec = {
-  content: 'text*',
+  content: '(text | highlight_marker)*',
```

Comment:
Can we make this "inline*" instead? Also can we get rid of the explicit marks below?

Reply:
updated